testsuite/test-generate-zeppelin.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+source test-generate-common.sh
+source test-common.sh
+source test-config.sh
+
+__GenerateZeppelinStandardTests_BasicTests() {
+    local sparkversion=$1
+    local javaversion=$2
+    local zeppelinversion=$3
+    local zeppelinhome=$4
+
+    cp ../submission-scripts/script-${submissiontype}/magpie.${submissiontype}-spark-with-zeppelin magpie.${submissiontype}-spark-with-zeppelin-${zeppelinversion}-${sparkversion}
+
```

Comment:
For consistency to other generated files, could we do:

`magpie.${submissiontype}-spark-with-zeppelin-${zeppelinversion}-spark-${sparkversion}`

so the second version number is clearly identified as a spark version.